### ISSUE TITLE
Some concerns about RAS development

### ISSUE THREAD

**Issue by ghost** [dillbyrne/random-agent-spoofer#583]
I know it's been said many times but i'm really worried about the future.

I use Firefox 52 ESR and I really hope someone (like gitbugged who proposed an unofficial 0.9.5.7 version with new features) or some people will keep RAS development maintained before 59 ESR version comes out.

Especially for script injections that no other addon offers from what i'm aware of.

**Comment by forkoz** [comment]
the api to do these things isn't available most likely.

**Comment by gitbugged** [comment]
Yes, that is my understanding as well - many of the API needed to do this addon won't be possible in the next version. There is some work that was done by @mylainos to [port](https://github.com/mylainos/random-agent-spoofer) the addon to the new WebExt framework, but he appears to have gone inactive.

Dark times for the free internet. Keep coding!

**Comment by neroux** [comment]
@forkoz It would require a rewrite for certain parts, but the interfaces are there. There are quite a few such extensions for Chrome right now.

**Comment by ghost** [comment]
[These 3 scripts](https://github.com/ghacksuserjs/ghacks-user.js/wiki/4.2.1-User-Scripts) are working on FF 57, two of them replace the window.name and history length scripts in RAS.
